guide.md

```diff
@@ -9779,15 +9897,15 @@ I prefer PrivaZer because it has more customization and smarter features, but I
 
 Both these tools can be used for cleaning many things such as:
 
--   The Windows USN journal which stores plenty of information[^460].
+-   The Windows USN journal which stores plenty of information.
 
 -   The Windows System Resource Usage Monitor (SRUM)[^461].
 
 -   Various histories of various programs (such as the recent lists).
 
 -   Various logs
 
--   The free (unallocated) space of your hard drive[^462].
+-   The free (unallocated) space of your hard drive][^462].

```

Comment:
Typo fix: ][, remove beginning closing bracket.

Reply:
![](https://i.postimg.cc/kX7WSDC2/2022-06-27-18-54.png)

That shows on my end. Note the bracket out of place.